README.adoc

```diff
@@ -444,6 +444,25 @@ Name of the docker image to build
 buildDockerImage('plugins-site-api')
 ----
 
+=== buildDockerImage_k8s
```

Comment:
```suggestion
=== buildDockerAndPublishImage
```

K8s is an implementation detail and rather belongs to docs IMHO

Reply:
its the k8s (infra, release, trusted?) version of jenkins. buildDockerImage is the one used in the non k8s versions of infra.

I suspect the difference will be short lived, as we probably don't need to run the shared library on other installs (its only used by wiki exporter right now, and ci.jenkins can probably just docker build and not worry about all the pushing and stuff)

Reply:
I was unable to figure out how to make the shared library work in both environments. Totally open to ideas. The limiting thing is kubernetes{} agent and container{}

https://github.com/jenkins-infra/pipeline-library/pull/156/files#diff-d62a5e7da2898a017f06db97b3974b15R16

Reply:
gonna rename, don't need both anymore